packages/venia-concept/src/components/Footer/footer.js

```diff
@@ -60,9 +60,9 @@ class Footer extends Component {
                         <span>Get answers from our community online.</span>
                     </p>
                 </div>
-                <div className={classes.copyright}>
-                    <span>© Magento 2018. All rights reserved.</span>
-                </div>
+                <small className={classes.copyright}>
```

Comment:
I was going to ask why this change relates to accessibility, and then I [checked](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small) and saw that:
> In HTML5, this element is repurposed to represent side-comments and small print, including copyright and legal text, independent of its styled presentation.
So this is a good change! 🥇